src/main/java/org/folio/circulation/domain/Item.java

```diff
@@ -189,11 +191,17 @@ private String mapToItemCallNumberPropertyName(String holdingsPropertyName) {
   }
 
   public ItemStatus getStatus() {
-    return ItemStatus.from(getStatusName());
+    ItemStatus itemStatus = ItemStatus.from(getStatusName());
+    itemStatus.setDate(getStatusDate());
```

Comment:
Why is this not passed in to the `from` method (which could be a constructor)?

Reply:
Done.